getting_started/step_by_step/scripting_player_input.rst

```diff
@@ -181,12 +181,12 @@ Summary
 
 In summary, every script in Godot represents a class and extends one of the
 engine's built-in classes. The node types your classes inherit from give you
-access to properties like ``rotation`` and ``position`` in our sprite's case.
+access to properties, such as ``rotation`` and ``position`` in our sprite's case.
 You also inherit many functions, which we didn't get to use in this example.
 
 In GDScript, the variables you put at the top of the file are your class's
 properties, also called member variables. Besides variables, you can define
-functions, which, for the most part, will be your classes' methods.
+functions, which, for the most part, will be your class's methods.
```

Comment:
This is supposed to be plural, the above line uses the singular, but here we're talking about classes in general

Reply:
Sure, fair enough. I think it works either way and can imagine arguments for both, but I'll go ahead and get this pull request changed (or submit a new one). Thanks for the feedback!